tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -25,19 +26,23 @@ const (
 
 	maxErrMsgLen = 1024
 
-	invalidExtraLabelsError = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidExtraLabelsError  = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidEvenExtraHeadersError = "invalid value for environment variable EXTRA_HTTP_HEADERS. Expected a comma separated list with an even number of entries."
+	invalidRegexExtraHeadersError = "invalid value for environment variable EXTRA_HTTP_HEADERS. Header key must conform with regex /[-A-Za-z0-9]+/ "
```

Comment:
can we use `httpHeaderKeyRegex.String()` here rather than hardcoding the regex into this error string, just in case the string for the regex changes in the future

Reply:
I removed the regex from the constant string and added `httpHeaderKeyRegex.String()` to the format string in the fmt.Errorf() call here: https://github.com/grafana/loki/pull/11883/commits/8d90c1003fed78eea21b12771125d2c0a3ec302a